.github/build.in.yml

```
@@ -160,7 +157,7 @@ jobs:
         with:
           linters: gcc
           run: |
-            bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
+            bash -o pipefail -c "lake exe mk_all || echo "There are unaccounted files, but I am not failing yet!"; env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
```

```suggestion
            bash -o pipefail -c "lake exe mk_all || echo "There are unaccounted for files, but I am not failing yet!"; env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
```
Otherwise looks good to me!
I am still thinking about a more drastic refactor which moves the style linters (including `mk_all`) out of build.yml but that can be done on top of this.